screen/ui-article-detail/src/main/java/kasem/sm/ui_detail/DetailVM.kt

```diff
@@ -32,7 +32,7 @@ class DetailVM @Inject constructor(
     savedStateHandle: SavedStateHandle,
 ) : ViewModel() {
 
-    private val articleId = savedStateHandle.get<String>("id")?.toInt() ?: -1
+    private val articleId = savedStateHandle.get<Int>("id") ?: -1
```

Comment:
I heard about this PR. Thanks for using compose destinations 🙂
You should avoid accessing nav args via their string key. Instead, use “savedStateHandle.navArgs<YourNavArgsClass>().id” for example 🙂

Reply:
Okay, thanks for the review, there are three occurrences like this in the this pr. I'll change them.

Reply:
Ah. Your library made that straightforward. That's great!! I'm inspired as well as amazed by the potential of this library.

Reply:
It’s just that compose destinations abstracts you from having to know how the args are handled. Also there are special cases where it may do more than just get it from savedStateHandle, and you may be surprised on how the info is saved in there, so it’s safer to always use the APIs 🙂

Reply:
Yup as I go through the changes in the code, I'm already realizing how this library can be helpful!!